web-common/src/features/dashboards/time-controls/utils/time-types.ts

```diff
@@ -0,0 +1,154 @@
+import type { DateTimeUnit } from "luxon";
+import type { V1TimeGrain } from "../../../../runtime-client";
+
+export const TIME = {
+  MILLISECOND: 1,
+  get SECOND() {
+    return 1000 * this.MILLISECOND;
```

Comment:
Can we refactor this to not use a getter? Feels like constants are better for performance.